test-it/ViaTests/EmptyVariantConstant_ComputeEqualsAndNotEquals_ReturnsExpected.via

```diff
@@ -0,0 +1,248 @@
+define (Application%3A%3AEmptyVariantConstant_ComputeEqualsAndNotEquals_ReturnsExpected%2Egviweb dv(.VirtualInstrument (
+    Locals: c(   // Data Space
```

Comment:
![emptyvariants](https://user-images.githubusercontent.com/2609749/57313474-4d502400-70b5-11e9-81a8-6ef0eee8aba2.PNG)

Additional tests involving clusters and arrays containing empty variants and wired to equals and not equals are written as scripted tests. See here - https://ni.visualstudio.com/DevCentral/_git/ASW/pullrequest/25582?_a=overview